stdlib/public/Concurrency/Actor.cpp

```diff
@@ -1411,7 +1441,7 @@ void DefaultActorImpl::enqueueStealer(Job *job, JobPriority priority) {
             "[Override] Scheduling a stealer for actor %p at %#x priority",
             this, newState.getMaxPriority());
         swift_retain(this);
-        scheduleActorProcessJob(newState.getMaxPriority());
+        scheduleActorProcessJob(newState.getMaxPriority()); // TODO: handle task executor?
```

Comment:
Need to make sure how enqueueStealer is used and if we need a way to propagate the task executor to it here.